main.ts

```diff
@@ -302,14 +302,36 @@ function openThisDamnFile(pathToVhaFile: string) {
         'finalObjectReturning',
         lastSavedFinalObject,
         pathToVhaFile,
-        globals.selectedOutputFolder + path.sep,   // app needs the trailing slash (at least for now)
+        getHtmlPath(globals.selectedOutputFolder),
         changedRootFolder,
         rootFolderLive
       );
     }
   });
 }
 
+/**
+ * Return an HTML string for a path to the `vha` file
+ * e.g. `C:\Some folder` becomes `C:/Some%20folder`
+ * @param anyOsPath
+ */
+function getHtmlPath(anyOsPath: string): string {
+      // Windows was misbehaving
+      // so we normalize the path (just in case) and replace all `\` with `/` in this instance
+      // because at this point Electron will be showing images following the path provided
+      // with the `file:///` protocol -- seems to work
+      const normalizedPath: string = path.normalize(anyOsPath);
+      const forwardSlashes: string = normalizedPath.replace(/\\/g, '/');
+      return forwardSlashes.replace(/ /g, '%20');
```

Comment:
I thought this didn't work on Windows - otherwise we could just use `encodeURI()`?

Reply:
The biggest problem is that the URL string is part of a conditional inside _ngStyle_ in a template:
```
[ngStyle]="{ 'background-image': somethingTrue ? oneURL : anotherURL }"
```
I guess it could be formatted beforehand in the template with `encodeURI()` too (I'll check).

Today again it's too late to investigate 😴 -- but I'll look into all places we're dealing with slashes (thank you for the https://github.com/whyboris/Video-Hub-App/commit/c37341c100b5514fe161d32e48e40534307c7829 ref) and see if I can consolidate things 😅